### PR TITLE
Fix #93 Adding Rubocop configuration

### LINKED ISSUE #93: Implement style guide based test using rubocop
[`Rubocop`](https://github.com/bbatsov/rubocop)is a great gem in Ruby for static code ananlyzer. 
Adding it as part of tests would insure that code follow language code standards.  

@navidshaikh Please add label `test` to this issue.

### PR DESCRIPTION
**Pull request by hferentschik**
Fix #93 
- Autocorrect with Rubocop and fixing simple violations
- Adjusting line length setting and adding rubocop_todo.yml

### CODE REVIEW

**Comment by hferentschik** [comment]
Basic rubocop setup addressing the obvious violations. Most of the remaining ones in the TODO config are about class/method length and complexity. These classes/methods need to be refactored now over time.

File: README.adoc [hunk 1]
```diff
@@ -215,6 +215,15 @@ list of available Rake tasks via:
 $ bundle exec rake -T
 ---------------------
 
+=== Code style
```
Comment (brgnepal) [inline README.adoc:218]: Something like 'Style tests' would be better IMO aligning with "Unit/acceptance tests".

File: README.adoc [hunk 2]
```diff
@@ -215,6 +215,15 @@ list of available Rake tasks via:
...
+
+The code style is enforced by http://batsov.com/rubocop[RuboCop]. To verify that your code
+against the rules, run:
```
Comment (brgnepal) [inline README.adoc]: To verify that your code follow the [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide), run the style test suite as:

File: README.adoc (hunk 1 above)
Comment (hferentschik) [inline README.adoc:218]: Except code style is not about tests/testing. This are a set of rules to be followed which might or might not be enforced.

File: README.adoc (hunk 2 above)
Comment (hferentschik) [inline README.adoc]: Jupp, let me reword this. I screwed this one up.

File: README.adoc (hunk 1 above)
Comment (brgnepal) [inline README.adoc:218]: @hferentschik "Style checks" ?

**Comment by hferentschik** [comment]
@budhrg ok, I changed the wording. I think it is better now.

File: README.adoc [hunk 3]
```diff
@@ -215,6 +215,19 @@ list of available Rake tasks via:
 $ bundle exec rake -T
 ---------------------
 
+=== Code style
+
+As most other open-source projects vagrant-service-manager has a set of conventions
```
Comment (brgnepal) [inline README.adoc]: @hferentschik  Comma needed after "As most other open-source projects" as doesn't read well in continuous with "vagrant-service-manager"

**Comment by brgnepal** [comment]
@hferentschik Added a comment for the need of comma.

File: README.adoc [hunk 4]
```diff
@@ -215,6 +215,19 @@ list of available Rake tasks via:
...
+about how to write code for it. It is much easier to understand a codebase when all the code in it is in
```
Comment (brgnepal) [inline README.adoc]: I don't think this line "It is much easier to understand a codebase when all the code in it is in a consistent style." is necessary.

Without this line, the description looks simple and enough.

**Comment by brgnepal** [comment]
LGTM. 

Merged and thanks.

**Pull request closed without merging**